src/lib/cli/command.js

```diff
@@ -50,8 +50,20 @@ args.argv = async function( argv, cb ): Promise<any> {
 	// Check for updates every day
 	updateNotifier( { pkg, isGlobal: true, updateCheckInterval: 1000 * 60 * 60 * 24 } ).notify();
 
+	// `help` and `version` are always defined as subcommands
+	const customCommands = this.details.commands.filter( c => {
+		switch ( c.usage ) {
+			case 'help':
+			case 'version':
+				return false;
+
+			default:
+				return true;
+		}
+	} );
+
 	// Show help if no args passed
-	if ( this.details.commands.length > 1 && ! this.sub.length ) {
+	if ( !! customCommands.length && ! this.sub.length ) {
```

Comment:
Why are there two different `bang` policies here, both on a `.length` params?

Reply:
It's a common Javascript way to cast something to a Boolean. The first casts the value to a Boolean, but inverts it, so the second one is needed to invert it back.

Reply:
We're checking to see if we have custom commands defined (`!! customCommands.length`; looking for `true`) and not passing any args (`! this.sub.length`; looking for `false`).

I think for clarity, this would be better written as:

```
if ( customCommands.length > 1
    && this.sub.length === 0 ) {
```

or take it a step further:

```
const haveCustomCommands = customCommands.length > 1;
const runningSubcommand = this.sub.length === 0;

if ( haveCustomCommands && ! runningSubcommand ) {
```